recipe/meta.yaml

```diff
@@ -29,18 +29,13 @@ requirements:
     - gmp
     - mpfr
     - qd
-    - fplll >=5.4.0
+    - fplll >=5.4.2
     - cysignals
     - numpy
     - python
     - pip
   run:
-    - gmp
-    - mpfr
-    - qd
-    - fplll
     - cysignals
-    - {{ pin_compatible('numpy') }}
```

Comment:
Why?

Reply:
numpy has a run exports. Is that not good enough?

Reply:
(and thanks for double checking btw.)

Reply:
I don't think we added the run-exports for the older versions.

Reply:
Ok. Then I'll revert that part.

Reply:
Anyway, it now fails because the fplll build had been broken on aarch64 it seems.